### PR TITLE
Refactor delete invitation

### PR DESCRIPTION
**Pull request by ANGIECATHERINEP**
Refactor delete invitation

### CODE REVIEW

**Review by LinconOssa — APPROVED** [review]
Meets the requirements.

File: src/pages/privileges/outlets/users/tabs/invitations/index.tsx [hunk 1]
```diff
@@ -7,19 +7,19 @@ import {
 import { useEffect, useState } from "react";
 import { EMessageType, IMessage } from "@src/types/messages.types";
 import { EAppearance } from "@src/types/colors.types";
-import { getAll } from "@mocks/utils/dataMock.service";
+import { deleteItemData, getAll } from "@mocks/utils/dataMock.service";
```
Comment (carin91) [inline src/pages/privileges/outlets/users/tabs/invitations/index.tsx:10]: Include spacing between import sections.

File: src/pages/privileges/outlets/users/tabs/invitations/index.tsx [hunk 2]
```diff
@@ -83,42 +83,24 @@ function InvitationsTab(props: InvitationsTabProps) {
     {
       id: "3",
```
Comment (carin91) [inline src/pages/privileges/outlets/users/tabs/invitations/index.tsx:84]: The id is 3 or Delete?

File: src/pages/privileges/outlets/users/tabs/invitations/index.tsx [hunk 3]
```diff
@@ -83,42 +83,24 @@ function InvitationsTab(props: InvitationsTabProps) {
...
       actionName: "Eliminar",
-      content: (invitation: IInvitationsEntry) => (
```
Comment (carin91) [inline src/pages/privileges/outlets/users/tabs/invitations/index.tsx:86]: Create the config file to include the actions for the table.
image

File: src/pages/privileges/outlets/users/tabs/invitations/DeleteInvitation/index.tsx [hunk 4]
```diff
@@ -1,25 +1,63 @@
 import { useState } from "react";
+import { functionById } from "@mocks/utils/dataMock.service";
```
Comment (carin91) [inline src/pages/privileges/outlets/users/tabs/invitations/DeleteInvitation/index.tsx:2]: Include the spacing between imports.

File: src/pages/privileges/outlets/users/tabs/invitations/DeleteInvitation/interface.tsx [hunk 5]
```diff
@@ -1,61 +1,75 @@
-import { DecisionModal } from "@components/feedback/DecisionModal";
-import { Button, Icon } from "@inube/design-system";
 import { MdOutlineDelete } from "react-icons/md";
-import { deleteInvitationModalConfig } from "../../../config/invitationsTable.config";
-import { useState } from "react";
+import { Icon, Stack } from "@inube/design-system";
+import { DecisionModal } from "@components/feedback/DecisionModal";
```
Comment (carin91) [inline src/pages/privileges/outlets/users/tabs/invitations/DeleteInvitation/interface.tsx:3]: Include the spacing between imports.

File: src/pages/privileges/outlets/users/tabs/invitations/DeleteInvitation/interface.tsx [hunk 6]
```diff
@@ -1,61 +1,75 @@
...
-            parentHover={isHovered ? true : false}
-            cursorHover
-          />
-        </div>
-      )}
+      <Stack
+        onMouseEnter={() => setHover(true)}
+        onMouseLeave={() => setHover(false)}
+      >
+        <Icon
```
Comment (carin91) [inline src/pages/privileges/outlets/users/tabs/invitations/DeleteInvitation/interface.tsx:47]: Something of the Icon is ill-defined. As you can see in the picture below, the icon is not centered.
image

**Review by carin91 — APPROVED** [review]

File: src/pages/privileges/outlets/users/tabs/invitations/index.tsx (hunk 1 above)
Comment (ANGIECATHERINEP) [reply src/pages/privileges/outlets/users/tabs/invitations/index.tsx]: resolved

File: src/pages/privileges/outlets/users/tabs/invitations/index.tsx (hunk 2 above)
Comment (ANGIECATHERINEP) [reply src/pages/privileges/outlets/users/tabs/invitations/index.tsx]: resolved

File: src/pages/privileges/outlets/users/tabs/invitations/index.tsx (hunk 3 above)
Comment (ANGIECATHERINEP) [reply src/pages/privileges/outlets/users/tabs/invitations/index.tsx:106]: resolved

File: src/pages/privileges/outlets/users/tabs/invitations/DeleteInvitation/index.tsx (hunk 4 above)
Comment (ANGIECATHERINEP) [reply src/pages/privileges/outlets/users/tabs/invitations/DeleteInvitation/index.tsx:3]: resolved

File: src/pages/privileges/outlets/users/tabs/invitations/DeleteInvitation/interface.tsx (hunk 5 above)
Comment (ANGIECATHERINEP) [reply src/pages/privileges/outlets/users/tabs/invitations/DeleteInvitation/interface.tsx:4]: resolved

File: src/pages/privileges/outlets/users/tabs/invitations/DeleteInvitation/interface.tsx (hunk 6 above)
Comment (ANGIECATHERINEP) [reply src/pages/privileges/outlets/users/tabs/invitations/DeleteInvitation/interface.tsx:50]: resolved

**Pull request merged**